compose-builder/Makefile

```diff
@@ -926,14 +936,14 @@ taf-compose-perf: gen
 	cat gen-header docker-compose.yml > $(RELEASE_FOLDER)/taf/docker-compose-taf-perf$(NO_SECURITY)$(BUS)$(ARCH).yml
 
 run: gen
-	docker-compose -p edgex up -d $(SERVICES)
+	${DOCKER_COMPOSE}  -p edgex up -d $(SERVICES)
 
 pull: gen
-	docker-compose pull $(SERVICES)
+	${DOCKER_COMPOSE}  pull $(SERVICES)
 
 gen:
 	$(GROVE_CHECK) \
-	docker-compose -p edgex $(COMPOSE_FILES) config > docker-compose.yml
+	${DOCKER_COMPOSE}  -p edgex $(COMPOSE_FILES) ${GEN_COMMAND} > docker-compose.yml
```

Comment:
If we are still supporting the older docker-compose until EdgeX 3.0, shouldn't the gen command continue to use the old version?

Reply:
It is using the version of compose you have installed. Developer (us) that run `make build` still need to use docker-compose  1.27.2 as stated is stated in the README. Use can user which ever they want.

Reply:
It doesn't look that way.  Pretty much every time you install docker now, you get the compose sub-command added if you follow the docker.com instructions.  The makefile is written such that if assumes you are using docker-compose, and then if "docker compose" is found, it switches to that.  I think if you want this behavior you should assume "docker compose" is available, and then if the developer has installed a specific version of docker-compose, to use that instead.

Reply:
Nope, opposite. It look for `docker-compose` installed. If found it uses it. If not is tries `docker compose`.

Reply:
I have both installed. If always uses `docker-compose`. See my test instructions.

Reply:
Ok.  I misread it

```
ifeq (, $(shell which docker-compose))
```

Not obvious, but I get it now.